CHANGELOG-4.0.md

```diff
@@ -45,29 +44,51 @@
 - Changed `Phalon\Tag::textArea` to use `htmlspecialchars` to prevent XSS injection. [#12428](https://github.com/phalcon/cphalcon/issues/12428)
 - Changed `Phalon\Cache\Backend\*::get` to use only positive numbers for `lifetime`
 - Changed `Phalcon\Logger` to comply with PSR-3. The component has been rewritten to use adapters that alllow logging to different areas. The [#13438](https://github.com/phalcon/cphalcon/issues/13438)
+- Changed `paginate()` in the place of `getPaginate`. Added `previous` in the place of `before`. [#13492](https://github.com/phalcon/cphalcon/issues/13492)
 
 ## Removed
-- PHP < 7.0 no longer supported
-- Removed `Phalcon\Model::reset` [#12317](https://github.com/phalcon/cphalcon/issues/12317)
-- Removed deprecated `Phalcon\Cli\Console::addModules`
-- Removed deprecated `Phalcon\Debug::getMajorVersion`
-- Removed deprecated `Phalcon\Mvc\Model\Criteria::addWhere`
-- Removed deprecated `Phalcon\Mvc\Model\Criteria::order`
-- Removed deprecated `Phalcon\Validation\Validator::isSetOption`
-- Removed deprecated `Phalcon\Security::hasLibreSsl`
-- Removed deprecated `Phalcon\Security::getSslVersionNumber`
-- Removed `Phalcon\Http\RequestInterface::isSoapRequested` in favor of `Phalcon\Http\Request::isSoap`
-- Removed `Phalcon\Http\RequestInterface::isSecureRequest` in favor of `Phalcon\Http\RequestInterface::isSecure`
-- Removed `Phalcon\Validation\MessageInterface` and `Phalcon\Mvc\Model\MessageInterface` in favor of `Phalcon\Messages\MessageInterface`
-- Removed `Phalcon\Validation\Message` and `Phalcon\Mvc\Model\Message` in favor of `Phalcon\Messages\Message`
-- Removed `Phalcon\Validation\Message\Group` in favor of `Phalcon\Messages\Messages`
+- PHP < 7.2 no longer supported
+- Removed `xcache` support from adapters
+- Removed `apc` support from adapters (use `apcu`)
+- Removed `memcache` support from adapters (use `libmemcached`)
```

Comment:
Is there any issue/PR we can mention here and below for each of change log entry?